workflow/executor/k8sapi/client.go

```diff
@@ -69,6 +70,9 @@ func (c *k8sAPIClient) getPod(ctx context.Context) (*corev1.Pod, error) {
 	err := waitutil.Backoff(backoffOver30s, func() (bool, error) {
 		var err error
 		pod, err = c.clientset.CoreV1().Pods(c.namespace).Get(ctx, c.podName, metav1.GetOptions{})
+		if err != nil && strings.Contains(err.Error(), "unknown (get pods)") {
```

Comment:
can we create a func return `errWithHelp(err) err`
then we can 

```golang
return !errorsutil.IsTransientErr(err), errWithHelp(err)
````

Reply:
thank you, fixed 410c2141b5edd9b14972d92abdc6bd68aac07798

Reply:
can we put `strings.Contains(err.Error(), "unknown (get pods)")` into own func? e.g. `isErrUnknownGetPods`

Reply:
still need updating?